config/config.go

```diff
@@ -304,6 +304,12 @@ func validateAdapters(adapterMap map[string]Adapter, errs configErrors) configEr
 type Metrics struct {
 	Influxdb   InfluxMetrics     `mapstructure:"influxdb"`
 	Prometheus PrometheusMetrics `mapstructure:"prometheus"`
+	Disabled   DisabledMetrics   `mapstructure:"disabled_metrics"`
+}
+
+type DisabledMetrics struct {
+	// True if we want to stop collecting account-specific metrics
```

Comment:
This comment may be misleading. If I understand the PR correctly, we're still collecting account-specific metrics, but no longer account-adapter specific metrics - that is, metrics for each adapter per account.

Reply:
That's true, let me change that